tensorboard/webapp/runs/views/runs_table/filterbar_component.ts

```diff
@@ -56,12 +58,18 @@ export class FilterbarComponent {
     return this.filters.get(this.selectedFilterName);
   }
 
+  constructor(private readonly customModal: CustomModal) {}
+
+  ngAfterViewChecked() {
+    this.customModal.runChangeDetection();
+  }
+
   openFilterMenu(event: MouseEvent, filterName: string) {
     this.selectedFilterName = filterName;
     const rect = (
       (event.target as HTMLElement).closest('mat-chip') as HTMLElement
     ).getBoundingClientRect();
-    this.filterModal.openAtPosition({
+    this.customModal.createAtPosition(this.filterModalTemplate, {
```

Comment:
One relatively minor bug I noticed:

1. Click on filter chip. Filter opens. (Good.)
2. Click on filter chip again. Filter closes. (Probably Good.)
3. Click on filter chip again. Nothing happens.  (Probably not Good?)

Reply:
Oh great catch. There was some residual logic pertaining to the old implementation that was hiding the modal on the second click (`*ngIf="selectedFilter"`). Removing this makes everything work as expected.